tests/unit_tests/blas/extensions/omatcopy2.cpp

```diff
@@ -185,6 +187,8 @@ TEST_P(Omatcopy2Tests, ComplexSinglePrecision) {
 }
 
 TEST_P(Omatcopy2Tests, ComplexDoublePrecision) {
+    CHECK_DOUBLE_ON_DEVICE(std::get<0>(GetParam()));
```

Comment:
Why are these needed only now?

Reply:
This has always been necessary, but it was missing. It skips the test if the device doesn't support double precision otherwise the test fails. It wasn't catch before because tests always ran on devices that supports it.